src/adapters/analytics/AnalyticsAdapter.js

```diff
@@ -0,0 +1,91 @@
+import CONSTANTS from 'src/constants.json';
+import { loadScript } from 'src/adloader';
+
+const events = require('src/events');
+const utils = require('../../utils');
+
+const BID_REQUESTED = CONSTANTS.EVENTS.BID_REQUESTED;
+const BID_TIMEOUT = CONSTANTS.EVENTS.BID_TIMEOUT;
+const BID_RESPONSE = CONSTANTS.EVENTS.BID_RESPONSE;
+const BID_WON = CONSTANTS.EVENTS.BID_WON;
+
+var _timedOutBidders = [];
+
+export default function AnalyticsAdapter({ global, url, handler }) {
+  var _queue = [];
+  var _eventCount = 0;
+  var _enableCheck = true;
+
+  loadScript(url, _emptyQueue);
+
+  return {
+    track: _track,
+    enqueue: _enqueue,
+    enableAnalytics: _enable
+  };
+
+  function _track({ eventType, args }) {
+    window[global](handler, eventType, args);
+  }
+
+  function _enqueue({ eventType, args }) {
+    const _this = this;
+
+    if (global && window[global] && eventType && args) {
+      _track({ eventType, args });
+    } else {
+      _queue.push(function () {
+        _eventCount++;
+        _this.track({ eventType, args });
```

Comment:
There seems to be a problem capturing `this` properly. When it's invoked via the script load callback, it's undefined.
![image](https://cloud.githubusercontent.com/assets/1870166/15784379/3b6a5456-2980-11e6-9795-cd1b6bf5e0ff.png)